cache/src/main/java/io/envoyproxy/controlplane/cache/Resources.java

```diff
@@ -38,8 +34,8 @@
 public class Resources {
```

Comment:
There's a static inner class called `V2` that perhaps cal also be removed?

Reply:
I must have missed it in the first pass, removed now